source/docs/software/getting-started/driver-station-best-practices.rst

```diff
@@ -0,0 +1,71 @@
+3. Consider used laptops rather than new. The FRC Driver Station and dashboard software uses very few system resources, so you don't need to buy a new laptop -- instead, buy a cheap 4-5 year old used one. You might even get one donated by a used computer store in your area.
+4. Laptop recommended features
+
+   a. RAM -- 2GB of RAM is minimum, if you have a SSD.
+   b. A display size of 13" or greater, with minimum resolution of 1440x1050.
+   c. Ports
+
+      i. A built-in Ethernet port is highly preferred. Ensure that it's a full-sized port. The hinged Ethernet ports don't hold up to repeated use.
+      ii. Use an Ethernet port saver to make your Ethernet connection. This extends the life of the port on the laptop. This is particularly important if you have a consumer-grade laptop with a hinged Ethernet port.
+      iii. If the Ethernet port on your laptop is dodgy, either replace the laptop (recommended) or buy a USB Ethernet dongle from a reputable brand. Many teams find that USB Ethernet is less reliable than built-in Ethernet, primarily due to cheap hardware and bad drivers. The dongles given to rookies in the KOP have a reputation for working well.
+      iv. 2 USB ports minimum
+
+   d. A keyboard. It’s hard to quickly do troubleshooting on touch-only computers at the field.
+   e. A solid-state disk (SSD). If the laptop has a rotating disk, spend $50 and replace it with a SSD.
+   f. Updated to the current release of Windows 10. Windows 10 is the most common OS now seen at competitions, so bugs are more likely to be found than on Windows.
+
+5. Install all Windows updates a week before the competition. This allows you time to ensure the updates will not interfere with driver station functions. To do so, open the Windows Updatesettings page and see that you're up-to-date. Install pending updates if not. Reboot and check again to make sure you’re up to date.
+6. Change "Active Hours" for Windows Updates to prevent updates from installing during competition hours. Navigate to Start -> Settings -> Update & Security -> Windows Update, then select Change active hours. If you’re traveling to a competition, take time zone differences into account. This will help ensure your driver station does not reboot or fail due to update installing on the field.
+7. Remove any 3rd party antivirus or antimalware software. Instead, use Windows Defender on Windows 10. Since you're only connecting to the internet for Windows and FRC software updating, the risk is low. Only install software on your driver station that's needed for driving.  Your goal here is to eliminate variables that might interfere with proper operation. Remove any unneeded preinstalled software ("crapware") that came with the machine. Don't use the laptop as your Steam machine for gaming back at the hotel the night before the event. Many teams go as far as having a separate programming laptop.
```

Comment:
Extra space
```suggestion
7. Remove any 3rd party antivirus or antimalware software. Instead, use Windows Defender on Windows 10. Since you're only connecting to the internet for Windows and FRC software updating, the risk is low. Only install software on your driver station that's needed for driving. Your goal here is to eliminate variables that might interfere with proper operation. Remove any unneeded preinstalled software ("crapware") that came with the machine. Don't use the laptop as your Steam machine for gaming back at the hotel the night before the event. Many teams go as far as having a separate programming laptop.
```